web/hooks/use-group.ts

```diff
@@ -200,7 +200,7 @@ export function useIsGroupMember(groupSlug: string) {
   const isAuthorized = useIsAuthorized()
   useEffect(() => {
     // if there is no user
-    if (isAuthorized === null) {
+    if (isAuthorized === false) {
```

Comment:
You could change `useIsAuthorized` to return a boolean. I think I intended that but made the change quickly.

Also acceptable: make it return undefined if the user/no user is not loaded.
Returning null makes no sense because that is another case of not authorized.